src/components/activities/Activity.jsx

```diff
@@ -137,21 +133,38 @@ const FullSizeCard = styled(Card)`
   overflow: auto;
 `;
 
-const ButtonWrapper = styled.div`
-  display: flex;
-
-  .button {
-    flex: 1;
-  }
-`;
-
 const TurnRevealWrapper = styled.div`
   .turnreveal-container {
     display: flex;
     justify-content: center;
   }
 `;
 
+const FlexContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  .title .location .price .button {
+    flex-shrink: 1;
```

Comment:
Why is this attribute necessary? I couldn't any difference.

Reply:
You're right, this was a left-over of another approach I tried.